new-website/deepchem/components/Layers/LayerCard.js

```diff
@@ -0,0 +1,59 @@
+import Image from "next/image";
+import Link from "next/link";
```

Comment:
This component is also very similar. I think some part has been omitted out, but I think can be reused.